src/views/studio/SignInJoinView/SignInProcessView/SignInProcessView.style.ts

```diff
@@ -22,23 +32,30 @@ export const SubTitle = styled(Text)`
 
 export const ListContainer = styled.div`
   max-width: 440px;
+  padding-bottom: 150px;
 `
 
 export const StyledCoinsIllustrations = styled(CoinsIllustration)`
   z-index: -2;
   bottom: 0;
-  left: 0;
-  width: 250px;
-  position: fixed;
+  position: relative;
   margin-top: 50px;
-  ${media.small} {
-    width: 300px;
+  width: 640px;
+  display: none;
+  ${media.large} {
+    left: -100px;
+    display: block;
```

Comment:
According to design coins illustration is visible on medium devices as well. We should include that, it will probably require some fiddling with position and size